### PR TITLE
F3: cart page

### PR DESCRIPTION
**Pull request by sushmita12345**
Added Cart Page screen to the E-commerce website.

Screenshot 2022-03-21 at 6 19 45 PM

### CODE REVIEW

File: src/pages/CartPage/Cart.css [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+  .horizontal-wrapper {
+    padding: 2rem;
+    display: flex;
+    width: 100%;
+  }
+  
+  .product-image {
+    width: 100%;
+    height: 100%;
+  }
```
Comment (devsurajrai) [inline src/pages/CartPage/Cart.css:214]: Remove commented code if not needed.

File: src/pages/CartPage/Cart.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                </div>
+                <button class="order-btn">PLACE ORDER</button>
+              </div>
+            </div>
+          </div>
+        </div>
+      ) : (
+        <h1 className="cart-empty-msg">Opps! Your Cart is Empty 😌</h1>
+      )}
+    </div>
```
Comment (devsurajrai) [inline src/pages/CartPage/Cart.js:141]: This can be broken down into a component and that can be rendered here.

**Review by devsurajrai — APPROVED** [review]
Really appreciate your work. Keep Going. Though try to implement the changes suggested.

File: src/pages/CartPage/Cart.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+import { MdiCurrencyInr, MdiClose } from "../../assets/Icon/icon";
+
+export function Cart() {
+
+  const {
+    filterState: { cart},
+    filterDispatch
+  } = useProduct();
+
+  console.log(cart);
```
Comment (aiman-ansari) [inline src/pages/CartPage/Cart.js:13]: You can remove console log here.

**Review by aiman-ansari — APPROVED** [review]
- Very neat and clean code only thing is that it would be easy to understand if you break your component into smaller peaces .
- Remove all the commented code.
- Do changes which are commented.

**Review by saurabhchirde — APPROVED** [review]
Didn't find any issue other than those already given suggestions.
Looks good to merge.

**Comment by sushmita12345** [comment]
Thank you, all changes has been done.

**Pull request merged**